modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -1157,7 +1179,10 @@ class DeConvolutionLayerImpl CV_FINAL : public BaseConvolutionLayerImpl
 #ifdef HAVE_INF_ENGINE
         if (backendId == DNN_BACKEND_INFERENCE_ENGINE)
         {
-            if (INF_ENGINE_RELEASE >= 2018050000 && (adjustPad.height || adjustPad.width))
+            if (kernel_size.size() == 3)
+                return false;
+
+            if (INF_ENGINE_RELEASE >= 2018050000 && (adjust_pads[0] || adjust_pads[1]))
```

Comment:
We can still check `adjustPad.height` and `adjustPad.width` and `dilation.width` with `dilation.height` here (2D convolution).

Reply:
done